test/Serilog.Sinks.AzureTableStorageWithProperties.Tests/Serilog.Sinks.AzureTableStorageWithProperties.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
Again change from multi-target to single target, i would suggest we keep net45 and update netcoreapp1.1 to netcoreapp3.1

Reply:
added netcoreapp3.1